src/dune_rules/ordered_set_lang.mli

```diff
@@ -92,5 +92,33 @@ module Unexpanded : sig
 end
 with type expanded := t
 
+module Include_dirs : sig
```

Comment:
This suggestion is more subjective so feel free to ignore/delay it:

It's a little weird that a general purpose thing such as `Ordered_set_lang` should depend on `Include_dir`. Therefore, I would expect `Include_dirs` to live in a separate module, or at least as a submodule of `Include_dir`.

Reply:
I agree in principle, but making that change would require exposing more of the inner working of `Ordered_set_lang` to the outside world, and that feels like a bigger change.